quiz/views.py

```diff
@@ -111,3 +115,11 @@ def _get_checked_answers(saved_answers, questions):
         elif str(pk) in saved_answers:
             checked_answers = checked_answers + saved_answers[str(pk)]
     return checked_answers
+
+
+def _compute_result(quiz, score):
```

Comment:
Maybe you could refactor this function since it was designed to work only if you have 3 or more predefined results attached to quiz (for 0 it will break with 500 error, for 1-2 the 'if' statement is not reached).
I think the function should handle all cases.

Also maybe you could define the score range score as a lower bound instead of upper bound. For example you could handle the cases as such based on number of score ranges defined:
* 0 : no message is shown no matter the score
* 1 :  < score_range: no message; > score_range: show message
* 2 :  < score_range1: no message;  between score_range1 and 2: show message1; > score_range2: show message2